libse/Utilities.cs

```diff
@@ -684,7 +684,9 @@ public static string UnbreakLine(string text)
             if (!text.Contains(Environment.NewLine))
                 return text;
 
-            var singleLine = text.Replace(Environment.NewLine, " ");
+            text = text.Replace('\r', ' ');
+            text = text.Replace('\n', ' ');
+            var singleLine = text;

```

Comment:
We shouldn't assume that text will ways contains use \r\n (Carriage-Return+Line-Feed) as line break character.